pages/community/tsc.js

```diff
@@ -225,7 +226,14 @@ function QuestionCard() {
         src="/img/avatars/questionmark.webp"
         className="mx-auto rounded-full h-20 w-20 xl:w-28 xl:h-28"
       />
-      <div className="my-4">Become a member!</div>
+      <div className="my-4">
+        Want to become a member?
+        Follow this
+        <Link href='https://www.youtube.com/watch?v=uG_aLF9Z1F0'>
```

Comment:
Please, use `TextLink` component https://github.com/asyncapi/website/blob/master/components/typography/TextLink.js, along with `target="blank"` to open the video in new tab.